Amplify/DevMenu/DevEnvironmentInfo.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+// struct to decode/encode information about developer environment in json format
+struct DevEnvironmentInfo: Codable {
+    let nodejsVersion, npmVersion, amplifyCLIVersion, podVersion: String
+    let xcodeVersion, osVersion: String
```

Comment:
prefer one declaration per line. We usually avoid declaring multiple fields per line.